src/resources/app-en.json

```diff
@@ -802,8 +802,8 @@
   "noRespondentIdentifier": "No respondent identifier yet.",
   "noRespondents": "No Respondents yet.",
   "noParticipants": "No Participants yet.",
-  "noRespondentsForApplet": "No Respondents yet. Invite your first one by clicking the ‘Add Respondent’ button above.",
-  "noParticipantsForApplet": "No Participants yet. Invite your first one by clicking the ‘Add Participant’ button above.",
+  "noRespondentsForApplet": "This applet has no respondents",
+  "noParticipantsForApplet": "This applet has no participants",
```

Comment:
_Cette applet n'a aucun participant_

Reply:
[Updated](https://github.com/ChildMindInstitute/mindlogger-admin/pull/1645/commits/c1213d76cc2ae4f1f3ac7771596b13e8eeefc399)!

Reply:
For now, yes